src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
 public class Application {
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery[] lotterys = new Lottery[]{
+                new Lottery().getRandomBall(),
+                new Lottery().getRandomBall(),
+                new Lottery().getRandomBall()
+        };
```

Comment:
That's not wrong. But not efficient. The same lottery may produce multiple Balls. So we don't need to create Lottery object for each ball. 
Imagine you have card deck and you want to play poker with your friends. With such approach, you don't want to reuse the same deck for the second game or the third one. You said to your friends "No, this deck is only for one game. Let's go buy another deck for the second game".

Reply:
I understand what u mean but i guess i do not understand how to create this code so it will work that way. Would be nice if u show me an example that I can use later in my projects

Reply:
> I understand what u mean but i guess i do not understand how to create this code so it will work that way. Would be nice if u show me an example that I can use later in my projects @MAMentorFX

Just create an object of Lottery, put it in variable. Create for-loop which iterates 3 times. Inside your loop call sout(lottery.getRandomBall().toString())

Reply:
omg i always think about more complicated stuff

Reply:
THX!!